modules/apps/bookmarks/bookmarks-service/build.xml

```diff
@@ -20,4 +20,10 @@
 			service.test.dir="../bookmarks-test/test/integration"
 		/>
 	</target>
+
+	<target name="build-upgrade-table">
+		<ant dir="../../../../portal-impl/" target="build-upgrade-table">
+			<property name="module.name" value="bookmarks" />
+		</ant>
+	</target>
```

Comment:
The whole thing looks fine. However, this one commit I think will not pass Brian's review mostly in light of the fact that it's not very "SDK" friendly. Notably, I think we'll need to support this operation for plugins outside of core modules and so we'll have to have some function to call... which we would use here.

I'm checking to see who I should maybe forward this too.